cds_dojson/marc21/fields/books/base.py

```diff
@@ -143,7 +151,11 @@ def authors(self, key, value):
     item = build_contributor_books(value)
     if item and item not in _authors:
         _authors.append(item)
-
+    try:
+        if 'u' in value and clean_val('u', value, str) == 'et al.':
```

Comment:
Is it possible that `et al.` can take different forms, like `et al` or `etc.`?

Also, won't `et al.` also be added twice with the `build_contributor_books` call?

Reply:
@equadon what do you mean twice? in the build_contributors there is no second creation of `et al.`